automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/methodrepo/MethodRepoSpec.scala

```diff
@@ -10,7 +10,7 @@ import org.broadinstitute.dsde.workbench.service.test.{CleanUp, WebBrowserSpec}
 import org.scalatest._
 
 
-class MethodRepoSpec extends FreeSpec with MethodFixtures with UserFixtures with WorkspaceFixtures with BillingFixtures
+class MethodRepoSpec extends FreeSpec with ParallelTestExecution with MethodFixtures with UserFixtures with WorkspaceFixtures with BillingFixtures
```

Comment:
hmm did I forget to disable parallel for this Method test??? I don't remember why.